src/SFA.DAS.AssessorService.Application/Handlers/ExternalApi/DataSync/ExternalApiDataSyncHandler.cs

```diff
@@ -1,51 +1,48 @@
-﻿using Dapper;
-using SFA.DAS.AssessorService.ExternalApiDataSync.Infrastructure;
-using SFA.DAS.AssessorService.ExternalApiDataSync.Logger;
-using System;
+﻿using System;
```

Comment:
Where is this file - it should be int he Application Library? - Ok -it's moved and renamed